### ISSUE TITLE
Incompatibility with KDE

### ISSUE THREAD

**Issue by Maddremor** [lacymorrow/crossover#100]
I saw that incompatibility with KDE was discovered in #77. Is this just thing one has to accept, or is there a will/intention/possibility to make it work? Switching your DE is not something you just do casually.

**Comment by lacymorrow** [comment]
Unfortunately it's an issue with Electron, not with CrossOver, so there's not much I can do to fix it from my end. Chances are it will just "be fixed" in a future update

**Comment by Maddremor** [comment]
I understand. Do you mind leaving this issue open for other people with the same issue to find? (Also for me to get a notification on an eventual resolution)

**Comment by lacymorrow** [comment]
Yep, this will stay open until it's fixed. When I have some time in the next few days I'll see about finding the bug/fix

**Comment by bunkbail** [comment]
+1 this as well. I was the one who filed the issue on #77. I just can't live with another DE. KDE is just too good now. Hoping that you can find a fix for this.

**Comment by bunkbail** [comment]
Oh btw, forgot to mention, crossover isn't compatible with KDE, it's incompatible with Kwin. You can still use KDE if you're using a different windows manager like Mutter or XFWM.
